apps/fabric-website/src/pages/Overviews/GetStartedPage/docs/mac/GetStartedOverview.md

```diff
@@ -1,11 +1,11 @@
-Fluent UI macOS is an [open-source](https://github.com/microsoft/fluentui-apple) native library that provides the Fluent UI experience for the macOS platform. It contains custom controls and customizations for platform controls, all from the official Fluent design language used in Microsoft 365 products.
+Fluent UI Apple is an [open-source](https://github.com/microsoft/fluentui-apple), native library used by Microsoft 365 apps. We offer customizations and custom controls that deliver unique Fluent experiences to customers using macOS devices.
```

Comment:
It wasn't clear to me if we still wanted open-source to be a link from the feedback.

Reply:
I would keep it as a link for now, but we can follow-up with Jennifer on the ADO item.